scripts/gen_ocabundlesjson.sh

```diff
@@ -34,13 +34,32 @@ fi
 echo -e "{" >${OCAIDSJSON}
 echo -e "[" >${OCALISTJSON}
 
+declare -a schema_files
+schema_files=(OCABundles/schema/*)
```

Comment:
Use readarray instead of declare -a to read an array from a command output. For example, instead of declare -a schema_files; schema_files=(OCABundles/schema/*), you can use readarray -t schema_files < <(find OCABundles/schema -type f).

```
schema_files=()
readarray -t schema_files < <(find OCABundles/schema -type f)
```